### ISSUE TITLE
Supported Stacks

### ISSUE THREAD

**Issue by dineshkummarc** [ProcessManagement/ProcessManagement#21]
Supported Stacks

Windows 7 (IIS, Dotnet, SQL Server)
Windows 7 (IIS, Dotnet, MySQL)
Windows 7 (Apache2, Dotnet, SQL Server)
Windows 7 (JBOSS, Dotnet, SQL Server)
Windows 7 (Apache2, Dotnet, MySQL)
Windows 7 (JBOSS, Dotnet, MySQL)
Windows 2012 (Apache2, Dotnet, SQL Server)
Windows 2012 (JBOSS, Dotnet, MySQL)
Windows 2012 (IIS, Dotnet, SQL Server)
Windows 2012 (IIS, Dotnet, MySQL)
Windows 8 (IIS, Dotnet, SQL Server)

**Comment by dineshkummarc** [comment]
Expending the numbers for manual stacks dotnet version wise

Dotnet v3.5 Stacks

E7DotNet35IDS Windows 7 (IIS, Dotnet, SQL Server)
E7DotNet35IDM Windows 7 (IIS, Dotnet, MySQL)
E7DotNet35ADS Windows 7 (Apache2, Dotnet, SQL Server)
E7DotNet35JDS Windows 7 (JBOSS, Dotnet, SQL Server)
E7DotNet35ADM Windows 7 (Apache2, Dotnet, MySQL)
E7DotNet35JDM Windows 7 (JBOSS, Dotnet, MySQL)
E12DotNet35ADS Windows 2012 (Apache2, Dotnet, SQL Server)
E12DotNet35JDM Windows 2012 (JBOSS, Dotnet, MySQL)
E12DotNet35IDS Windows 2012 (IIS, Dotnet, SQL Server)
E12DotNet35IDM Windows 2012 (IIS, Dotnet, MySQL)
E8DotNet35IDS Windows 8 (IIS, Dotnet, SQL Server)

Dotnet v4 .0 Stacks

E7DotNet4 0IDS Windows 7 (IIS, Dotnet, SQL Server)
E7DotNet4 0IDM Windows 7 (IIS, Dotnet, MySQL)
E7DotNet4 0ADS Windows 7 (Apache2, Dotnet, SQL Server)
E7DotNet4 0JDS Windows 7 (JBOSS, Dotnet, SQL Server)
E7DotNet4 0ADM Windows 7 (Apache2, Dotnet, MySQL)
E7DotNet4 0JDM Windows 7 (JBOSS, Dotnet, MySQL)
E12DotNet4 0ADS Windows 2012 (Apache2, Dotnet, SQL Server)
E12DotNet4 0JDM Windows 2012 (JBOSS, Dotnet, MySQL)
E12DotNet4 0IDS Windows 2012 (IIS, Dotnet, SQL Server)
E12DotNet4 0IDM Windows 2012 (IIS, Dotnet, MySQL)
E8DotNet4 0IDS Windows 8 (IIS, Dotnet, SQL Server)

Dotnet v4 .5 Stacks

E7DotNet4 5IDS Windows 7 (IIS, Dotnet, SQL Server)
E7DotNet4 5IDM Windows 7 (IIS, Dotnet, MySQL)
E7DotNet4 5ADS Windows 7 (Apache2, Dotnet, SQL Server)
E7DotNet4 5JDS Windows 7 (JBOSS, Dotnet, SQL Server)
E7DotNet4 5ADM Windows 7 (Apache2, Dotnet, MySQL)
E7DotNet4 5JDM Windows 7 (JBOSS, Dotnet, MySQL)
E12DotNet4 5ADS Windows 2012 (Apache2, Dotnet, SQL Server)
E12DotNet4 5JDM Windows 2012 (JBOSS, Dotnet, MySQL)
E12DotNet4 5IDS Windows 2012 (IIS, Dotnet, SQL Server)
E12DotNet4 5IDM Windows 2012 (IIS, Dotnet, MySQL)
E8DotNet4 5IDS Windows 8 (IIS, Dotnet, SQL Server)

**Comment by denno-secqtinstien** [comment]
so you wanna say as we would be moving up for the dotnet version we also would be having the updated stacks for us ?

**Comment by dineshkummarc** [comment]
yes Ofcourse, if we are using dotnet 3.0 then stacks based on 3.0, if we are using dotnet3.5 then stacks based on 3.5 and so on. all configurations would be different for different version.
